packages/store/ts/config/v2/compat.ts

```diff
@@ -71,5 +71,5 @@ export function storeToV1<store>(store: conform<store, Store>): storeToV1<store>
     codegenIndexFilename: store.codegen.indexFilename,
     tables: resolvedTables,
     v2: store,
-  } as unknown as storeToV1<store>;
+  } as never;
```

Comment:
ooc why `as never`?

Reply:
Especially if you have to cast to `unknown` first, the cast itself provides no type-safety.

The return type is already explicitly specified on the function's definition, so `as never` is just a convenient way to allow an assignment to a predefined type (in this case `storeToV1<store>`) without duplicating the signature.

It would also be fine to use `as any` in this position, but `as never` is more likely to surface downstream if the explicit signature is ever accidentally removed.

Also less likely to trigger linting rules 😄